superset-embedded-sdk/src/index.ts

```diff
@@ -77,6 +80,7 @@ export async function embedDashboard({
   id,
   supersetDomain,
   mountPoint,
+  iframeTitle = t("Embedded Dashboard"),
```

Comment:
Hey this look like a public interface, so this new [optional] param should be the last one in the function call for backwards compatibility, otherwise we'l have off-by-one param matching on the function call!